src/Raven.Server/Documents/Queries/Parser/QueryParser.cs

```diff
@@ -454,8 +461,18 @@ private bool Operator(out QueryExpression op)
                     };
                     return true;
                 case OperatorType.Method:
-                    return Method(field, op: out op);
+                    var method = Method(field, op: out op);
+
+                    if (method && Operator(false, out var methodOperator))
```

Comment:
This piece of code makes queries like `count() > 100` are parsed properly. Note that in this case both: `count()` and `>` are operators: `OperatorType.Method` and `OperatorType.GreaterThan`